### PR TITLE
Added some colors I need for implementing typography changes in unit …

### PR DESCRIPTION
**Pull request by BrianGenisio**
I added a couple of colors that were referenced in the design: zpl://screen?sid=5a9f4afe2066820f0cfc0356&pid=59dec10a2179d16778ca1a5c

### CODE REVIEW

File: packages/wonder-blocks-color/index.js [hunk 1]
```diff
@@ -18,4 +20,6 @@ const Color = {
     salmon: "#FD6C6E",
 };
 
+Color.azul = Color.blue;
```
Comment (jeresig) [inline packages/wonder-blocks-color/index.js]: I think this is a mistake - it should just be "blue" (azul is blue in spanish, I don't think we necessarily need this!)

File: packages/wonder-blocks-color/index.js [hunk 2]
```diff
@@ -8,6 +8,8 @@ const Color = {
 
     // Neutral
     offBlack: "#21242C",
+    offBlackSecondary: "rgba(33, 36, 44, 0.5)",
+    offBlackBorder: "rgba(33, 36, 44, 0.1)",
```
Comment (jeresig) [inline packages/wonder-blocks-color/index.js]: I don't think either of these colors match what we have in the style guideline for black:

screenshot 2018-03-21 15 35 26

Comment (BrianGenisio) [reply packages/wonder-blocks-color/index.js]: Ah, that is brand new, I think.  I didn't know they approved it.

**Comment by BrianGenisio** [comment]
@jeresig I just confirmed with @tatianasalazar  that this color has been approved, so I've updated this PR.

**Comment by jeresig** [comment]
Great, thank you for bouncing this back and making the changes! I left a comment on the design spec to see if we can get clarification about the 10% vs. 16%, but no reason to hold up this change - so go ahead!

**Comment by alopatin** [comment]
I'm sorry to comment after this is already merged, but is there any chance that we could use declarative names for our color variables? e.g. `offBlack64` instead of `offBlackSecondary` (this is going to be our secondary text color, but also our main modal backdrop color... and will likely gain many more uses as we go along).

**Comment by BrianGenisio** [comment]
@alopatin I had it that way before I created the PR as, `offBlack50` and `offBlack10`, but realized that using the numbers for this is coupling the implementation of the color with the name of the color, which is a maintenance issue.  For example, when we changed `offBlack50` to `offBlack64`, I'd need to change every place in the code which uses it.  But when I use something like `offBlackSecondary`, I change it in Wonerblocks once, and the entire site gets changed automatically.

If you don't like `Secondary`, is there a better descriptive name we can use?  Perhaps something a little more fuzzy like `offBlackPale` and `offBlackGhost` for 64 and 10 respectively?

**Comment by alopatin** [comment]
@BrianGenisio My understanding is that we plan to version `wonder-blocks-color`, just like we'll be versioning `wonder-blocks-button` and all of the other wonder blocks packages/components. In that world, this sort of tight coupling isn't an issue. The whole point of versioning is so that we _don't_ have to change the entire site at once. This is definitely tricky right now; we don't have the import semantics fully established and some packages are more solid than others. But I think it's the right approach long term.

The other thing I'm trying to keep in mind is that ideally, the variables names we use in code should match the variables names designers use in Sketch (which we then see in Zeplin). Right now, all of the grays I'm seeing in Zeplin are marked "off-black-XX".

**Comment by BrianGenisio** [comment]
@alopatin FWIW, the designs I'm seeing are different.  It isn't `off-black-XX` but it is `dark-XX`.  It is easy enough to change in a separate diff... we just need to make sure we have a clear answer to what the color will be.  @jeresig do you have any insight?

**Comment by BrianGenisio** [comment]
I chatted with Taty and Nefaur.  They are still working on the naming convention for colors, so let's hold off until the've solidified it.  They are converging on a convention like `Off-Black (64%)` which would likely map to @alopatin's `offBlack64` suggestion.  They are hoping to have it codified by the end of the week.

**Pull request merged**